src/format.test.ts

```diff
@@ -0,0 +1,17 @@
+import { Design, Display, Pillar, Special } from './format';
+
+it('Design enum contains Article', () => {
+	expect(Design.Article).toBeDefined();
+});
+
+it('Display enum contains Standard', () => {
+	expect(Display.Standard).toBeDefined();
+});
+
+it('Pillar enum contains News', () => {
+	expect(Pillar.News).toBe(0);
+});
+
+it('Special enum contains SpecialReport', () => {
+	expect(Special.SpecialReport).toBe(5);
+});
```

Comment:
This keeps coveralls happy but I'm not sure that it's much use beyond that?

Reply:
Ah, interesting that coveralls is really keen for you to run tests on what are essentially just type definitions - there's not really any logic to test there.

That said, testing the `Theme` enum might be useful, it's built as a compound of two other `enum`s so I guess there's a chance for something to get broken there.

Otherwise, yeah I agree, this seems reasonable just to keep coveralls happy, but it's probably just running over ground that TSC covers anyway.

Reply:
They're actually good tests to have. It's not unfeasible that someone might modify the type and accidentally modify an existing Type due to a typo, or  change the ordering and have that somehow break something